### PR TITLE
hotline deployment conf and yaml as well as added to jenkins

### PR DESCRIPTION
**Pull request by jtorano-va**
Related Story: https://vasdvp.atlassian.net/browse/API-1241

### CODE REVIEW

File: products/hotline.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: $NAMESPACE
```
Comment (jhulbert-va) [inline products/hotline.yaml]: In the deployer, this needs to be `$DU_NAMESPACE` (see hotline.conf)

File: products/hotline.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: $NAMESPACE
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: salesforce-ingress
+  namespace: $NAMESPACE
```
Comment (jhulbert-va) [inline products/hotline.yaml]: Same as above.

File: products/hotline.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: $DU_NAMESPACE
```
Comment (jhulbert-va) [inline products/hotline.yaml]: Where be the rest of your metadata? (Check out exemplar.)

File: products/hotline.conf [hunk 4]
```diff
@@ -0,0 +1,8 @@
+DU_ARTIFACT=health-apis-hotline-deployment
+DU_VERSION=1.0.1
+DU_NAMESPACE=views-hotline
+DU_DECRYPTION_KEY="$CRYPTO_KEY"
+DU_HEALTH_CHECK_PATH="/patsr/healthz"
+DU_LOAD_BALANCER_RULES[1]="/api/patsr*"
```
Comment (bschofield-va) [inline products/hotline.conf]: I think these should be `*/patsr/*`, `*/sfdc/*` and `*/vimt/*`. Does the application support both `/patsr/<stuff>` and `/api/patsr/<different-stuff>`? It does per your ingress below.

File: products/hotline.yaml (hunk 3 above)
Comment (bschofield-va) [reply products/hotline.yaml]: Make sure this is forward looking to the S3 support...
```
metadata:
  name: $DU_NAMESPACE
  labels:
    deployment-id: $K8S_DEPLOYMENT_ID
    deployment-unit: $PRODUCT
    deployment-unit-artifact: $DU_ARTIFACT
    deployment-unit-version: $DU_VERSION
    deployment-date: $BUILD_DATE
    deployment-s3-bucket: $DU_AWS_BUCKET
    deployment-s3-folder: $DU_S3_FOLDER
```

**Pull request merged**